src/v2/Components/Authentication/EmailSubscriptionCheckbox.tsx

```diff
@@ -0,0 +1,29 @@
+import { Serif } from "@artsy/palette"
+import Checkbox from "v2/Components/Checkbox"
+import React from "react"
+import styled from "styled-components"
+
+export const EmailSubscriptionCheckbox = ({
+  error,
+  name,
+  onChange,
+  onBlur,
+  value,
+  ...props
+}) => {
+  const color = error && !value ? "red100" : "black60"
```

Comment:
**issue [blocking]:** remove red highlight

I suspect there's a copy/paste mistake here - I _think_ we should remove this line from this component because the label should not be highlighted in red. But I also think we can simplify the `SignUpForm` component and will suggest over there too. Have you tried this in the browser? I wonder if `error` is ever true?

*format via [Conventional Comments](https://conventionalcomments.org/)*

Reply:
Tried this in browser, and it never seems to show the red. I am going to remove it.

Reply:
Removed this.